app/controllers/director/counselors_controller.rb

```diff
@@ -29,15 +29,15 @@ def create
       )
       if @counselor.save
         puts "COUNSELOR SAVED"
-        redirect_to counselor_path(:id => @counselor.id)
+        redirect_to counselors_path
```

Comment:
You changed it? Isn't it supposed to be `counselor_path(:id => @counselor.id)` ?